Adds APIM pages docs (#3868)
wso2/docs-apim: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-alerts.md

# Alerts Page

> This page illustrates information related to business alerts issued by the system for your currently selected

Reviewer: Can we explain, 'system' and 'whole team'?

Suggestion:
"This page shows the information related to business alerts issued by the system for your currently selected
 environment and organization. You can use this page as a health monitoring dashboard and make it visible to 
the whole team. You can use this page to drill down on each alert and discover possible anomalies or errors in your published APIs. 

See the topics given below to get details of the available widgets and understand how to use them to best effect."

Author: fixed with cc3b796

 environment and organization. You can use this page as a health monitoring dashboard and make it visible to whole
  team. Using this page you can drilldown on each alert and discover possible anomalies or errors in your published
   APIs. Following sections of the document will guide you through the available widgets and how to use them to best
    effect.
    
## Prerequisites

> By default, API analytics doesn't have any alerts enabled. Hence for you to use this page you should configure alerts

Reviewer: ```suggestion
Be sure to configure alerts if you want to use this page. By default, alerts are not enabled for API analytics. 
```

>  and then should have API invocations which will trigger alerts.

Reviewer: ```suggestion
```

# Alerts Page

> This page illustrates information related to business alerts issued by the APIM analytics for your currently selected

Reviewer: ```suggestion
This page shows information related to business alerts issued by APIM analytics for your currently selected
```
